analyzers/stylecop.json

```diff
@@ -0,0 +1,19 @@
+{
+  "$schema": "https://raw.githubusercontent.com/DotNetAnalyzers/StyleCopAnalyzers/master/StyleCop.Analyzers/StyleCop.Analyzers/Settings/stylecop.schema.json",
+  "settings": {
+    "orderingRules": {
+      "systemUsingDirectivesFirst": true,
+      "usingDirectivesPlacement": "outsideNamespace",
+      "elementOrder": [
+        "constant",
+        "static",
+        "accessibility",
+        "readonly",
+        "kind"
```

Comment:
Open question: Does it make sense to have `kind` configured when we can't use SA1201 (yet)?